pkg/pipelines/pipelines.go

```diff
@@ -0,0 +1,111 @@
+package pipelines
+
+import (
+	"log"
+	"testing"
+	"time"
+
+	"github.com/openshift-pipelines/release-tests/pkg/client"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+	tb "github.com/tektoncd/pipeline/test/builder"
+	"gotest.tools/v3/icmd"
+)
+
+const (
+	TaskName1                 = "create-file"
```

Comment:
Can we have a better name  ( e.g. 'CreateFileTask` ) than `TaskName1` especially because it is exported?